.github/workflows/external-integrations-unit-tests.yml

```
@@ -0,0 +1,41 @@
+name: EXTERNAL INTEGRATIONS unit tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch'
+        required: true
+        default: 'main'
+  pull_request:
+    paths:
+      - '.github/workflows/external-integrations-unit-tests.yml'
+      - 'integrations/**'
+      - 'src/Makefile'
+
+jobs:
+  build: 
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.10']
+    env:
+      PYTHONPATH: /home/runner/work/wazuh/wazuh/api:/home/runner/work/wazuh/wazuh/framework
+    steps:
+      - uses: actions/checkout@v3
```

We could take advantage of the fact that we are adding/updating all unit test workflows to update the version of this action to v4

```suggestion
      - uses: actions/checkout@v4
```

Same in the rest of the files